code/modules/shuttle/shuttle.dm

```diff
@@ -11,19 +11,48 @@
 
 	resistance_flags = RESIST_ALL
 	anchored = TRUE
-//
+
+	/// The identifier of the port or ship.
+	/// This will be used in numerous other places like the console,
+	/// stationary ports and whatnot to tell them your ship's mobile
+	/// port can be used in these places, or the docking port is compatible, etc.
```

Comment:
```suggestion
	/** 
	  * The identifier of the port or ship.
	  * This will be used in numerous other places like the console,
	  * stationary ports and whatnot to tell them your ship's mobile
	  */ port can be used in these places, or the docking port is compatible, etc.
```